### PR TITLE
bring back new conversation list item on desktop

### PR DESCRIPTION
**Pull request by zanderz**
@keybase/react-hackers

### CODE REVIEW

File: shared/chat/inbox/index.desktop.js [hunk 1]
```diff
@@ -175,6 +129,7 @@ class Inbox extends PureComponent<Props, State> {
             onSelectUp={this.props.onSelectUp}
             onSelectDown={this.props.onSelectDown}
           />
+          // TODO: there is no longer showNewConversation
```
Comment (MarcoPolo) [inline shared/chat/inbox/index.desktop.js]: This shows up! 
screen shot 2017-11-01 at 10 27 15 am

Comment (zanderz) [reply shared/chat/inbox/index.desktop.js]: HA! whoops!

**Comment by buoyad** [comment]
I just picked up [this](https://keybase.atlassian.net/browse/DESKTOP-5379), and in investigating the issue, it looks like something in [this range](https://github.com/keybase/client/compare/37963d3c464d27c73b61ced2c3689150a19b19b5...master?w=1) broke the new-conversation-with-avatar row getting hooked up to the inbox. 

I'm trying to figure out the cause now, but maybe we should wait on merging this one in the meantime?

**Comment by zanderz** [comment]
@buoyad @MarcoPolo how bout a thumb?

**Comment by buoyad** [comment]
@zanderz check out the behavior when starting a new conversation on current master. I think the list item has been brought back already by another recent change.

**Comment by zanderz** [comment]
I just tried master, the original ticket is to show "New Conversation" in the list instead of the participant names, which still appear on master, so this seems to still be needed

**Comment by buoyad** [comment]
Ah my mistake, I thought this was a regression that was fixed in a drive-by at some point. Reviewing now

File: shared/chat/inbox/row/new-conversation.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+class NewConversation extends PureComponent<{}> {
+  render() {
+    return (
+      <div
+        style={{
+          ...globalStyles.flexBoxRow,
+          alignItems: 'center',
+          backgroundColor: globalColors.blue,
+          flexShrink: 0,
+          minHeight: 48,
```
Comment (buoyad) [inline shared/chat/inbox/row/new-conversation.js]: the other conversations have `height: 56` as do the designs

Comment (zanderz) [reply shared/chat/inbox/row/new-conversation.js]: ah, that part was just moved from existing, can change

**Comment by zanderz** [comment]
ok I didn't understand that part, I double checked with Ceceile - will try to fix that

**Comment by zanderz** [comment]
OK, Cecile set me straight, I think it works the right way now @keybase/react-hackers @buoyad

**Comment by chrisnojima** [comment]
i don't think this works correctly. if you have existing conversations this shows up and you no longer see the old conversation

**Pull request merged**